### PR TITLE
Improve cairo rs

### PR DESCRIPTION
**Pull request by Juan-M-V**

### CODE REVIEW

File: cairo_fuzz/fuzz/src/main.rs [hunk 1]
```diff
@@ -1,80 +1,104 @@
...
+    io::Write,
+    path::PathBuf,
 };
-use cairo_vm::types::program::Program;
-use cairo_vm::vm::vm_core::VirtualMachine;
-use cairo_vm::cairo_run::write_output;
-use cairo_vm::cairo_run::CairoRunConfig;
-
 
+#[derive(Arbitrary)]
```
Comment (daphneherlambda) [inline cairo_fuzz/fuzz/src/main.rs]: arbitrary is not in use so this is unnecesary, maybe it is for later?

**Review by daphneherlambda — CHANGES_REQUESTED** [review]
LGTM just a little thing and we´re done

File: cairo_fuzz/fuzz/src/main.rs (hunk 1 above)
Comment (Juan-M-V) [reply cairo_fuzz/fuzz/src/main.rs]: Resolved: https://github.com/lambdaclass/fuzzing_examples/pull/18/commits/cdeb521ce7c3709a8d286489798652cde7b25e4a & https://github.com/lambdaclass/fuzzing_examples/pull/18/commits/80cd5d9859c94095c5abb13a1d2b872ad4221e4e

**Review by daphneherlambda — APPROVED** [review]

**Review by fmoletta — APPROVED** [review]

**Review by Oppen — COMMENTED** [review]
Looks good. Do note that in cairo-rs there's a pending PR that extracts the logic from `cairo_vm_cli::main` into `cairo_vm_cli::run(args: impl Iter<Item = String>) -> Result<_>`, so I can make it public if you need to avoid the duplication we see in here.

**Pull request closed without merging**